app/javascript/mastodon/components/admin/Dimension.jsx

```diff
@@ -74,7 +72,7 @@ export default class Dimension extends PureComponent {
                 </td>
 
                 <td className='dimension__item__value'>
-                  {typeof item.human_value !== 'undefined' ? item.human_value : <FormattedNumber value={item.value} />}
+                  <span title={item.human_value}>{item.value}</span>
```

Comment:
Displaying the raw value is not particularly helpful for space usage:
![image](https://github.com/mastodon/mastodon/assets/384364/1104e920-76e9-4251-a8e3-1cb8b765fc0d)